prow/plugins/branchcleaner/branchcleaner.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
2018

Reply:
Done.

That comment could be made for a lot of the files in this repo :P Should kubernetes  projects update all their headers every year or how is this handled?

Reply:
> Should kubernetes projects update all their headers every year or how is this handled?

The year reflects the year the file was created/added. :)